crates/release_plz_core/src/git/backend.rs

```diff
@@ -422,7 +560,7 @@ impl GitClient {
         })?;
 
         let prs = match self.backend {
-            BackendType::Github => {
+            BackendType::Github | BackendType::Gitlab => {
```

Comment:
are you sure this works?
The response returned by [this](https://docs.gitlab.com/ee/api/commits.html#list-merge-requests-associated-with-a-commit) endpoint doesn't seem compatible with GitPR

Reply:
btw I reviewed the rest of the code and done some few changes. It looks great!
I just need to test this once we resolve this comment and then we should be done 👍

Reply:
I was also surprised this worked right away and did not investigate it further. But you are right, this doesn't look sound.

Reply:
Well spotted, I don't know why this worked. But the decision whether or not to build a release worked correctly and still does after the fix now :-)